var/spack/repos/builtin/packages/nvidia-hpc-sdk/package.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from spack.util.prefix import Prefix
+import platform
+import os
+
+
+class NvidiaHpcSdk(Package, CudaPackage):
+    """The NVIDIA HPC Software Development Kit (SDK) includes the proven compilers,
+       libraries and software tools essential to maximizing developer productivity
+       and the performance and portability of HPC applications."""
+
+    homepage = "https://developer.nvidia.com/hpc-sdk"
+
+    maintainers = ['ajkotobi']
+
+    version('2020_207', sha256='a5c5c8726d2210f2310a852c6d6e03c9ef8c75e3643e9c94e24909f5e9c2ea7a')
+    #Probably better way to deal with this
+    if platform.machine() == "aarch64":
+	variant(
+          'cuda', default='11.0', description='List of CUDA that are enabled',
+          values=('11.0'), multi=False)
+    else:
+        variant(
+          'cuda', default='11.0', description='List of CUDA that are enabled',
+          values=('11.0', '10.2', '10.1'), multi=False)
+
+    variant(
+        'network', default='network', description='Network installation',
+        values=('network', 'single'), multi=False
+    )
+
+    depends_on('cuda@11.0.2:',   when='cuda=11.0')
+    depends_on('cuda@10.2.89:',  when='cuda=10.2')
+    depends_on('cuda@10.1.243:', when='cuda=10.1')
```

Comment:
With regards to the CUDA stuff, I would remove the variant and instead use:
```python
depends_on('cuda')
conflicts('^cuda@:10', when='platform=aarch64')
```
This allows the user to control the version of CUDA used from the command-line, like:
```console
$ spack install nvidia-hpc-sdk ^cuda@10.2.89
```